error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -375,6 +377,19 @@ Flux<T> after(Flux<T> flux) {
     }
   }
 
+  /** Prefer {@link reactor.util.context.Context#empty()}} over more verbose alternatives. */
+  static final class ContextEmpty {
+    @BeforeTemplate
+    Context before() {
+      return Context.of(ImmutableMap.of());
```

Comment:
Users outside of Picnic might be more likely to use `new HashMap<>()`; let's add a case for that.

Reply:
(Better would be to introduce a `Matcher` for this, but that's something for a separate PR.)